### ISSUE TITLE
Abnormal circulating AXL Receptor Tyrosine Kinase concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5989]
**Preferred term label:** Abnormal circulating AXL Receptor Tyrosine Kinase concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of AXL receptor tyrosine kinase in the blood is outside the normal range., PMID: 27350258

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating AXL Receptor Tyrosine Kinase concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of AXL receptor tyrosine kinase  in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating AXL Receptor Tyrosine Kinase concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
No normal ranges have been determined for this protein in serum. For instance, there is no LOINC term for this test. Therefore, it is possible to do a cohort study that shows a higher level in individuals with some condition, but it is not possible to do a test in an individual and say that the level is "too high" or "too low". This means that this would not be a valid HPO term.
